src/thor/trace_route_action.cc

```diff
@@ -178,11 +179,11 @@ thor_worker_t::map_match(Api& request) {
     m_temp_disconnected_edges.clear();
 
     // Form the path edges based on the matched points and populate disconnected edges
-    m_temp_path_edges = MapMatcher::FormPath(matcher.get(), match_results, edge_segments,
```

Comment:
i took the liberty to revert some of the premature optimization that went on some PRs ago. it has no measurable impact on performance and hurts readibility (hungarian notation is known to do that :) )